### PR TITLE
VACMS-1031 poc dash events et

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #1031 

## Testing done
Visual

## Screenshots
image

## QA steps
 - As an admin, go to `pittsburgh-health-care/events` and review listings.
 - Play with filters, and make sure things sort and filter as expected

### CODE REVIEW

File: tests/behat/drupal-spec-tool/views.feature [hunk 1]
```diff
@@ -16,7 +16,6 @@ Feature: Views
 | Recent content | content_recent | Content | Disabled | Recent content. |
 | Content served from Drupal | content_served_from_drupal | Content | Enabled | An exportable list of all content served from Drupal |
 | Content Entity Reference Source | content_entity_reference_source | Content | Enabled |  |
-| Custom block entity browsers | custom_block_entity_browsers | Custom Block | Enabled | For placing on content forms |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/views.feature:19]: This seems unrelated.  What's up with this?

Comment (ethanteague) [reply tests/behat/drupal-spec-tool/views.feature:19]: I just assumed it was related to some work someone else was doing, @swirtSJW - with latest db and master, running behat dst was telling me I needed this view in my test.

Comment (swirtSJW) [reply tests/behat/drupal-spec-tool/views.feature:19]: Nope... that sounds off.  If someone elses PR removed that, then their PR would have failed.   Check with Kev to see if he killed off entity browser on prod.

Comment (ethanteague) [reply tests/behat/drupal-spec-tool/views.feature:19]: Oh.....I see, @swirtSJW - it's just being moved to the bottom of the test. It's still there. Apologies - I don't remember why / how it was moved in the dst sheet, but it's still there.

**Pull request merged**